### PR TITLE
Removed unused Model params

### LINKED ISSUE #5297: Unused parameter : "model" in ReportLastScan methods
most methods of **ReportLastScan.java** has a parameter:
`org.parosproxy.paros.model.ModelModel model` , but it is actually unused. those methods just pass this parameter to: `public void generate(StringBuilder report, Model model) throws Exception` where it is not used at all.

### PR DESCRIPTION
**Pull request by Flo4l**
Removed unused Model params in `ReportLastScan.java` methods.

Closes #5297

Signed-off-by: Florian Albers <Florian.Albers@adesso.de>

### CODE REVIEW

File: zap/src/main/java/org/parosproxy/paros/extension/report/ReportLastScan.java [hunk 1]
```diff
@@ -84,18 +85,18 @@ public ReportLastScan() {}
...
     @Deprecated
-    public File generate(String fileName, Model model, String xslFile) throws Exception {
+    public File generate(String fileName, String xslFile) throws Exception {
         StringBuilder sb = new StringBuilder(500);
-        this.generate(sb, model);
+        this.generate(sb);
         return ReportGenerator.stringToHtml(sb.toString(), xslFile, fileName);
     }
 
-    public File generate(String fileName, Model model, ReportType reportType) throws Exception {
```
Comment (thc202) [inline zap/src/main/java/org/parosproxy/paros/extension/report/ReportLastScan.java]: We need to keep binary compatibility, for the add-ons.

Comment (Flo4l) [reply zap/src/main/java/org/parosproxy/paros/extension/report/ReportLastScan.java]: Force pushed required changes.

Comment (thc202) [reply zap/src/main/java/org/parosproxy/paros/extension/report/ReportLastScan.java]: Sorry I was not clear, the comment applies to the other public methods as well. We should deprecate the public methods with unused parameter and add new ones without the parameter (the initial commit was just missing the old/deprecated methods).

Comment (Flo4l) [reply zap/src/main/java/org/parosproxy/paros/extension/report/ReportLastScan.java]: My bad. Should be fine now.

File: zap/src/main/java/org/parosproxy/paros/extension/report/ReportLastScan.java [hunk 2]
```diff
@@ -87,15 +88,24 @@ public ReportLastScan() {}
      *     Model model, ReportType reportType)}
      */
     @Deprecated
-    public File generate(String fileName, Model model, String xslFile) throws Exception {
```
Comment (thc202) [inline zap/src/main/java/org/parosproxy/paros/extension/report/ReportLastScan.java]: This change should be reverted (the linked method in the JavaDoc could point to the new method as before).

File: zap/src/main/java/org/parosproxy/paros/extension/report/ReportLastScan.java [hunk 3]
```diff
@@ -125,7 +135,7 @@ private static File generateReportWithXsl(String report, String reportFile, Stri
         }
     }
 
-    public void generate(StringBuilder report, Model model) throws Exception {
```
Comment (thc202) [inline zap/src/main/java/org/parosproxy/paros/extension/report/ReportLastScan.java]: This needs to be kept/deprecated.

**Comment by thc202** [comment]
Thanks!

**Pull request merged**